### PR TITLE
Auth UI v2 - Alternative signup methods

### PR DESCRIPTION
**Pull request by IniZio**
ref #3597

**Signup**

Screenshot 2024-01-23 at 18-26-19 Authgear
Screenshot 2024-01-23 at 18-26-09 Authgear

**Login**

Screenshot 2024-01-23 at 18-16-11 Authgear
Screenshot 2024-01-23 at 18-16-03 Authgear

### CODE REVIEW

File: authui/src/authflowv2/components/divider.css [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @layer components {
   :root {
     /* FIXME(davis): Design use opacity on it which should be not necessary */
-    --divider__bg-color: var(--color-grey-200);
+    --divider__bg-color: var(--color-grey-100);
```
Comment (tung2744) [inline authui/src/authflowv2/components/divider.css:4]: oh when was it changed..

File: authui/src/authflowv2/components/secondary-button.css [hunk 2]
```diff
@@ -11,7 +11,7 @@
     );
     --secondary-btn__font-weight: var(--typography-label-large__font-weight);
     --secondary-btn__ring-width: 1.5px;
-    --secondary-btn__ring-color: var(--color-grey-100);
+    --secondary-btn__ring-color: #f0f1f1;
```
Comment (tung2744) [inline authui/src/authflowv2/components/secondary-button.css:14]: Should use grey 100 here.
Screenshot 2024-01-24 at 11 47 12 AM

File: pkg/auth/handler/webapp/viewmodels/authflow_branch.go [hunk 3]
```diff
@@ -106,6 +106,8 @@ func newAuthflowBranchViewModelStepAuthenticate(screen *webapp.AuthflowScreenWit
 			addChannelBranch(idx, o)
 		case config.AuthenticationFlowAuthenticationSecondaryOOBOTPSMS:
 			addChannelBranch(idx, o)
+		case config.AuthenticationFlowAuthenticationSecondaryTOTP:
```
Comment (tung2744) [inline pkg/auth/handler/webapp/viewmodels/authflow_branch.go:109]: :shocked: looks like a bug exist in main too

File: authui/src/authflowv2/components/secondary-button.css (hunk 2 above)
Comment (IniZio) [reply authui/src/authflowv2/components/secondary-button.css:14]: Looks like the alternative login button's have different style then, confirming with designer.

Comment (tung2744) [reply authui/src/authflowv2/components/secondary-button.css:14]: I think designer pointed it to the incorrect grey-100 in the dark mode palette. We can keep using the original grey-100 first.

File: pkg/auth/handler/webapp/viewmodels/authflow_branch.go (hunk 3 above)
Comment (IniZio) [reply pkg/auth/handler/webapp/viewmodels/authflow_branch.go:109]: Yes, I will create another pr to main with this commit after.

Comment (IniZio) [reply pkg/auth/handler/webapp/viewmodels/authflow_branch.go:109]: I suspect we will catch more of these later on, but seems not worth trying to handle in https://github.com/authgear/authgear-server/issues/3653 🤔

File: authui/src/authflowv2/components/divider.css (hunk 1 above)
Comment (louischan-oursky) [reply authui/src/authflowv2/components/divider.css:4]: Is the FIXME still relevant?

Comment (IniZio) [reply authui/src/authflowv2/components/divider.css:4]: Should not be, let me remove it.

cc @Ngkaokis

Comment (IniZio) [reply authui/src/authflowv2/components/divider.css:3]: Removed in ee8c1c0ff99ca232a47710b44f871f4feb6e6e3b

File: pkg/auth/handler/webapp/viewmodels/authflow_branch.go (hunk 3 above)
Comment (IniZio) [reply pkg/auth/handler/webapp/viewmodels/authflow_branch.go:109]: Base branch changed to main

**Pull request merged**